gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Default to regional STS endpoints
```

Comment:
I think this can be an issue and not a feature. Also mention what option is changing (`sts_regional_endpoints`)